libcontainer/nsenter/nsexec.c

```diff
@@ -882,6 +982,18 @@ void nsexec(void)
 						sane_kill(stage1_pid, SIGKILL);
 						bail("failed to sync with child: write(SYNC_MOUNTSOURCES_ACK)");
 					}
+					break;
+				case SYNC_MOUNT_IDMAP_PLS:
+					write_log(DEBUG, "stage-1 requested to open idmap sources");
+					send_idmapsources(syncfd, stage1_pid, config.idmapsources,
+							  config.idmapsources_len);
+					s = SYNC_MOUNT_IDMAP_ACK;
+					write_log(DEBUG, "idmap sources done, sending SYNC_MOUNT_IDMAP_ACK");
```

Comment:
@rata A small suggestion: I think you used this log when you were debugging. If you want to remain this debug log, please add `stage-xxx` in the head. But I suggest to remove this log to keep the same log as `mount sources`.

Reply:
Sure, I deleted this one. Thanks for checking the consistency!

When we wrote the mount sources patch with a co-worker we didn't add any debug lines, but when writing this PR I did, so I created a PR to add them for mount sources too: https://github.com/opencontainers/runc/pull/3734/files (I wrote that PR before finishing this one, probably)

I'm on the fence if these lines are useful or not, but if they are, I'll open a new PR adding them to the idmap and mount sources cases.